### ISSUE TITLE
Service "" is not register with `::class` concatenation.

### ISSUE THREAD

**Issue by soullivaneuh** [lookyman/phpstan-symfony#14]
With this code line:

```
$consumer = $this->getContainer()->get('test.'.AnalysisConsumer::class);
```

I have this message:

```
Service "" is not registered in the container. 
```

But the service exists.

**Comment by lookyman** [comment]
I'll look into it, thanks.

**Comment by lookyman** [comment]
Fixed by 5d0da0e.

**Comment by soullivaneuh** [comment]
Thanks!
